card.go

```diff
@@ -170,3 +185,24 @@ func deepClone(ix interface{}) interface{} {
 	}
 	return ix
 }
+
+func fillNameComponentMap(m map[string]interface{}, items interface{}) {
+	for _, v := range items.([]interface{}) {
+		// This map always has a single key - wrapper so this is O(1) not O(n).
+		for _, v := range v.(map[string]interface{}) {
```

Comment:
I'd fix the variable clashing here. I think it's simpler to reason about the code without clashing. You can do e.g.:

```go
for _, item := range items.([]interface{}) {
    // This map always has a single key - wrapper so this is O(1) not O(n).
    for _, v := range item.(map[string]interface{}) {
```

You can use `i` as well but it's more idiomatic to use `i` for index. Or you can rename `items interface{}` to `entries interface{}` and use `e` as a pointer.

Reply:
Fair point. I am hesitant to go with `entries`, since we iterate over `items` on the Python side so it's easier to reason about when the naming matches. Instead, will go with:

```go
	for _, wrappedItem := range wrappedItems.([]any) {
		// This map always has a single key - wrapper so this is O(1) not O(n).
		for _, item := range wrappedItem.(map[string]any) {
			component := item.(map[string]any)
```